resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/KubernetesVolumeUtils.scala

```diff
@@ -16,9 +16,12 @@
  */
 package org.apache.spark.deploy.k8s
 
+import scala.util.Try
+
 import org.apache.spark.SparkConf
 import org.apache.spark.deploy.k8s.Config._
 
+
```

Comment:
Could you remove this addition? The existing one line looks okay to me. (https://github.com/databricks/scala-style-guide#blank-lines-vertical-whitespace)
> Use one or two blank line(s) to separate class or object definitions.

Reply:
yes! just waiting for the checks to pass. Removing now